client/db/benches/state_access.rs

```diff
@@ -16,19 +16,72 @@
 // You should have received a copy of the GNU General Public License
 // along with this program. If not, see <https://www.gnu.org/licenses/>.
 
-use criterion::{criterion_group, criterion_main, BatchSize, Criterion};
+use criterion::{black_box, criterion_group, criterion_main, measurement::WallTime, Criterion};
 use rand::{distributions::Uniform, rngs::StdRng, Rng, SeedableRng};
+use rayon::prelude::*;
 use sc_client_api::{Backend as _, BlockImportOperation, NewBlockState, StateBackend};
 use sc_client_db::{Backend, BlocksPruning, DatabaseSettings, DatabaseSource, PruningMode};
-use sp_core::H256;
+use sp_core::{twox_128, H256};
 use sp_runtime::{
 	testing::{Block as RawBlock, ExtrinsicWrapper, Header},
 	StateVersion, Storage,
 };
+use std::{
+	ffi::OsString,
+	path::{Path, PathBuf},
+	time::{Duration, Instant},
+};
 use tempfile::TempDir;
 
 pub(crate) type Block = RawBlock<ExtrinsicWrapper<u64>>;
 
+fn get_polkadot_storage_keys() -> Vec<Vec<u8>> {
+	// This is a dump of all of the keys from Polkadot block
+	// `ad2fedfc2e6da47289b41cc24617915d4bce0638ca41bd0c4e4864ebb869f2dd`.
+	let compressed_blob = include_bytes!("keys.bin.zst");
+	let blob = zstd::bulk::decompress(&compressed_blob[..], 153917787).unwrap();
+	let keys: Vec<Vec<u8>> = codec::Decode::decode(&mut &blob[..]).unwrap();
+	assert_eq!(keys.len(), 1872246);
+	keys
+}
+
+fn get_account_keys_prefix() -> Vec<u8> {
+	let mut prefix = Vec::new();
+	prefix.extend(twox_128(b"System"));
+	prefix.extend(twox_128(b"Account"));
+
+	assert_eq!(
```

Comment:
why do we need assert_eq! here? we're not testing twox_128 here, correct?

Reply:
This is mostly for documentation's sake. I could have left a comment, but why do that when I can leave an `assert`?